mbuild/compound.py

```diff
@@ -80,6 +83,25 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
              "distances and standard residue templates!")
         structure = pmd.load_file(filename, structure=True, **kwargs)
         compound.from_parmed(structure, coords_only=coords_only)
+
+    elif smarts:
+        pybel = import_('pybel')
+        # First we try trating filename as a SMARTS string
+        try:
+            mymol = pybel.readstring("smi", filename)
+        # Now we treat it as a filename
+        except(OSError,IOError):
+            # For now, we only support reading in a single SMARTS molecule,
+            # but pybel returns a generator, so we get the first molecule
+            mymol = next(pybel.readfile("smi", filename))
```

Comment:
Could you also raise a warning here that describes what you've put in your comment?